_maps/configs/syndicate_ngr_kaliandhi.json

```diff
@@ -0,0 +1,54 @@
+{
+	"faction": "/datum/faction/syndicate",
+	"prefix": "NGRV",
+	"namelists": [
+		"GORLEX",
+		"NATURAL_AGGRESSIVE",
+		"BEASTS",
+		"WEAPONS"
+	],
+	"map_name": "Kali Andhi-class Destroyer",
+	"map_short_name": "Kali Andhi-class",
+	"description": "A post-ICW destroyer design commissioned by the New Gorlex Republic and outfitted for general patrol and reclamation duties. The Kali Andhi-class, despite its young age, has been instrumental in securing New Gorlex's modest territorial claims, making them a somewhat unwelcome sight for the PGF's frontier mission.",
```

Comment:
You should add a fluff bit about what its named for

Reply:
It's just named after a windstorm (a pattern I want to repeat in other NGR ships to make the fact that it's a pattern... yknow, obvious)